akka-stream/src/main/boilerplate/akka/stream/FanOutShape.scala.template

```diff
@@ -20,12 +21,12 @@ object FanOutShape {
   }
 }
 
-abstract class FanOutShape[I] private (_in: Inlet[I], _registered: Iterator[Outlet[_]], _name: String) extends Shape {
+abstract class FanOutShape[-I] private (_in: Inlet[I @uncheckedVariance], _registered: Iterator[Outlet[_]], _name: String) extends Shape {
```

Comment:
All this `@uncheckedVariance` dance is needed because an Inlet/Outlet really wants use-site variance (because it does not have any methods on it) but expressing that involves additional type parameters or existential wildcards that are just not bearable (and sometimes even not expressible).